deploy/scripts/src/ipv-core/test.ts

```diff
@@ -363,12 +363,14 @@ export function drivingLicence (): void {
     { isStatusCode302 })
     res = timeRequest(() => {
       const response = http.get(env.ipvCoreURL + res.headers.Location, { tags: { name: 'B02_DrivingLicence_04_DCMAWContinue_02_CoreCall' } })
-      if (env.staticResources) getStaticResources(response)
+      // if (env.staticResources) getStaticResources(response)
       return response
     },
     { isStatusCode200, ...pageContentCheck('Enter your UK photocard driving licence details and answer security questions online') })
   })
 
+  sleepBetween(1, 3)
```

Comment:
```suggestion
  sleepBetween(0.5, 1)
```
Let's reduce the think times to reduce iteration time and hence reduce VU count and memory usage

Reply:
Done